anago

```diff
@@ -1107,72 +1225,144 @@ prepare_workspace () {
       done
     fi
     logecho -n "Removing/recreating $WORKDIR: "
-    logrun cd /tmp
+    logrun cd $TMPDIR
     logrun -s rm -rf $WORKDIR || return 1
     logrun mkdir -p $WORKDIR || return 1
   fi
 
   # Check if locally staged directory contains all of the RELEASE_VERSIONs for
   # this release type
-  if ! ((FLAGS_noclean)) && \
-     {((FLAGS_stage)) || ! is_staged_locally ${RELEASE_VERSION[@]};}; then
+  if ((FLAGS_stage)) || ! found_staged_location ${RELEASE_VERSION[@]}; then
     # Sync the tree
-    gitlib::sync_repo $K8S_GITHUB_SSH $TREE_ROOT || return 1
+    gitlib::sync_repo $K8S_GITHUB_AUTH_URL $TREE_ROOT || return 1
   fi
-
-  logrun cd $TREE_ROOT
 }
 
 
 ##############################################################################
 # Archive the release on GS
+# @param --files-only - non-recursive
 archive_release () {
+  local arg=$1
   local archive_bucket="gs://$RELEASE_BUCKET/archive"
   local build_dir=${WORKDIR##*/}
-
-  ((FLAGS_yes)) \
-   || common::askyorn -y "Archive this release on $archive_bucket" || return
```

Comment:
I usually skip this on mock runs. Is there a reason this is required now?

Reply:
I figured it was useful to just run by default and the user could ^C out of it if desired.  That may be less than ideal.   I'll put the prompt back.  Thanks for the feedback on this!